test/features/parser-markdown.ts

```diff
@@ -39,6 +39,31 @@ export const testMarkdownParser = () => {
       expect(parsed.body).toHaveProperty('children[0].children[0].tag', 'code-inline')
     })
 
+    test('Keep meta from fenced code block', async () => {
+      const parsed = await $fetch('/api/parse', {
+        method: 'POST',
+        body: {
+          id: 'content:index.md',
+          content: [
+          '``` ts [file.ts]{4-6,7} other code block info',
+          'let code = undefined;',
+          'return code;',
+          '```'
```

Comment:
```suggestion
            '```ts [file.ts]{4-6,7} other code block info',
            'let code = undefined;',
            'return code;',
            '```'
```